src/python/turicreate/toolkits/style_transfer/style_transfer.py

```diff
@@ -412,10 +412,10 @@ def create(style_dataset, content_dataset, style_feature=None,
         for key in mps_weights:
             if "transformer" in key and "conv" in key:
```

Comment:
Should the two if-statement be combined? The bodies of each if-statement now (I think) exactly the same.
The combined test could be `if "transformer" in key and ("inst" in key or "conv" in key):`

Reply:
Good catch. Thanks!